smallrye-reactive-messaging-rabbitmq/src/main/java/io/smallrye/reactive/messaging/rabbitmq/IncomingRabbitMQMessage.java

```diff
@@ -263,7 +263,7 @@ public io.vertx.mutiny.rabbitmq.RabbitMQMessage getRabbitMQMessage() {
     }
 
     @Override
-    public synchronized void injectMetadata(Object metadataObject) {
+    public synchronized void injectMetadata(Object metadata) {
```

Comment:
Wouldn't be better to retain the name of the existing parameter (`metadataObject`) and instead change the body i.e.

```java
public synchronized void injectMetadata(Object metadataObject) {
  this.metadata = this.metadata.with(metadataObject);
}
```

This avoids having a parameter name that shadows that of a class member.